lib/api/core/plugins/pluginsManager.js

```diff
@@ -22,7 +22,7 @@
 'use strict';
 
 const
-  debug = require('debug')('kuzzle:plugins'),
+  debug = require('../../../debug')('kuzzle:plugins'),
```

Comment:
I find it far too easy to forget that there is a `debug` source file in Kuzzle's directories, and another one as a node module.
I bet that before long we'll have `require('debug')` lines in our code, making our debug capabilities inconsistent.

To make things easier I have these proposals:

* rename `../../../debug` into `../../../kuzzleDebug`

or:

* add `debug` as a `kuzzle` property, making it accessible almost everywhere in the project without needing any require beforehand. That would force several changes throughout Kuzzle code, though.

or:

* I hesitate to propose this one, but we also could create a global `debug` constant